spec/src/main/asciidoc/interoperability.asciidoc

```diff
@@ -61,11 +61,10 @@ upn:: This MP-JWT custom claim is the user principal name in the java.security.P
     principal name in javax.security.enterprise.identitystore.IdentityStore. If this claim is missing, fallback to
     the http://openid.net/specs/openid-connect-core-1_0.html#StandardClaims["preferred_username", OIDC Section 5.1] should be
     attempted, and if that claim is missing, fallback to the "sub" claim should be used.
-groups:: This MP-JWT custom claim is the list of group names that have been assigned to the principal of the MP-JWT.
-    This typically will require a mapping at the application container level to application deployment roles, but a
-    one-to-one between group names and application role names is required to be performed in addition to any
-    other mapping.
 
+[NOTE] It is recommended that JWT tokens have a `groups` claim if the endpoint requires Authorization but MP JWT implementations can map the groups from the other claims if the tokens have been issued by OpenId Connect and other providers which currently do not support MP JWT. Mapper SPI for extracting the groups from the custom claims will be supported in the future versions of this specification in order to make MP JWT implementations more interoperable.
```

Comment:
I suggest to delete "Mapper SPI for extracting the groups from the custom claims will be supported in the future versions of this specification in order to make MP JWT implementations more interoperable." as it muddles the water and no point to put some future features in the current spec as we don't have control which future version it will end up with or maybe never.

Reply:
@Emily-Jiang sounds good